server/src/DbtTextDocument.ts

```diff
@@ -203,6 +203,14 @@ export class DbtTextDocument {
 
   getModelPath(): string {
     const index = this.rawDocument.uri.indexOf(this.workspaceFolder);
+    const filePath = this.rawDocument.uri.slice(index + this.workspaceFolder.length + 1);
+    if (filePath.startsWith('dbt_packages') || filePath.startsWith('dbt_modules')) {
+      return filePath
+        .replaceAll('/', '.')
+        .replace(/^(dbt_packages|dbt_modules)./, '')
+        .replace('models.', '')
```

Comment:
Models location could be overwritten in `dbt_project.yml`. Moreover models could be located in multiple directories. E.g. `source-paths: ["models", "other_models"]`.

Reply:
Let's think if it would be better to create separate utils function, e.g. `extractModelName`.

Reply:
`dbt_packages` and `dbt_modules` to consts?